File: Documents/GitHub/Web-Projects/project5/calc_trig.js
// Trig functions
export function sine(angle) {
    return Math.sin(angle);
}

export function cosine(angle) {
    return Math.cos(angle);
}

export function tangent(angle) {
    return Math.tan(angle);
}

const degs_90 = Math.PI/2;
const degs_180 = Math.PI;  
const degs_270 = 3*Math.PI/2; 
const degs_360 = 2*Math.PI; 
const degs_0 = 0; 

console.log(`PI/4: ${Math.PI/4}`); 
console.log(`PI: ${Math.PI}`); 
 
console.log(`cosine of ${degs_90}: ${sine(degs_90)}`); 
console.log(`cosine of ${degs_180}: ${sine(degs_180)}`); 
console.log(`cosine of ${degs_270}: ${sine(degs_270)}`); 
console.log(`cosine of ${degs_360}: ${sine(degs_360)}`); 
console.log(`cosine of ${degs_0}: ${sine(degs_0)}`);
 
console.log(`sine of ${degs_90}: ${sine(degs_90)}`); 
console.log(`sine of ${degs_180}: ${sine(degs_180)}`); 
console.log(`sine of ${degs_270}: ${sine(degs_270)}`); 
console.log(`sine of ${degs_360}: ${sine(degs_360)}`); 
console.log(`sine of ${degs_0}: ${sine(degs_0)}`);

console.log(`tangent of ${degs_90}: ${tangent(degs_90)}`); 
console.log(`tangent of ${degs_180}: ${tangent(degs_180)}`); 
console.log(`tangent of ${degs_270}: ${tangent(degs_270)}`); 
console.log(`tangent of ${degs_360}: ${tangent(degs_360)}`); 
console.log(`tangent of ${degs_0}: ${tangent(degs_0)}`); 

File: Documents/GitHub/Web-Projects/project2/calc_trig.rb
# Various trig functions
module CalcTrig
  # Function to calculate sine
  # @param angle [Numeric] The angle to find the sine of, entered in radians
  # @return [Float] The sine of the angle
  def sine(angle)
    Math.sin(angle)
  end

  # Function to calculate cosine
  # @param angle [Numeric] The angle to find the cosine of, entered in radians
  # @return [Float] The cosine of the angle
  def cosine(angle)
    Math.cos(angle)
  end

  # Function to calculate tangent
  # @param angle [Numeric] The angle to find the tangent of, entered in radians
  # @return [Float] The tangent of the angle
  def tangent(angle)
    Math.tan(angle)
  end

end